c/termination-recursive-malloc/chunk1_true-termination.c

```diff
@@ -0,0 +1,28 @@
+#include <stdlib.h>
+
+extern int __VERIFIER_nondet_int(void);
+
+void chunk(int *data) {
+	if(data[0] >= data[3]) 
+		return;
+	
+	data[0]++;

```

Comment:
hm, actually there cannot be an overflow, because data[3] has to be bigger than data[0].